src/ExtensionsSample/Samples/ApiHubTableSamples.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.ApiHub.Sdk.Table;
+using Microsoft.Azure.WebJobs;
+using Newtonsoft.Json.Linq;
+
+namespace ExtensionsSample.Samples
+{
+    // To use the ApiHubTableSamples:
+    // 1. Add an AzureWebJobsSql app setting or environnment variable for your ApiHub SQL connection. 
+    //    The format should be: Endpoint={endpoint};Scheme={scheme};AccessToken={accesstoken}
+    // 2. Call config.UseApiHub() in Program.cs
+    // 3. Add typeof(ApiHubTableSamples) to the SamplesTypeLocator in Program.cs
+    // 4. Create table1 in your SQL database:
+    //      CREATE TABLE table1
+    //      (
+    //          c1 int NOT NULL,
+    //          c2 nvarchar(10) NULL
+    //          CONSTRAINT PK_table1 PRIMARY KEY(c1)
+    //      )
+    public static class ApiHubTableSamples
+    {
+        public static void BindToTableClient(
+            [ApiHubTable("AzureWebJobsSql")]
```

Comment:
I think rather than having the connection name repeated on each binding, you should instead follow the pattern of the other bindings (e.g. DocumentDB https://github.com/Azure/azure-webjobs-sdk-extensions/blob/master/src/WebJobs.Extensions.DocumentDB/DocumentDBAttribute.cs#L65) which allows the default connection to be specified on the global config, and allows per attribute overrides via the attribute property.